src/schedule/zephyr_ll.c

```diff
@@ -252,6 +254,17 @@ static void zephyr_ll_run(void *data)
 		       NOTIFIER_TARGET_CORE_LOCAL, NULL, 0);
 }
 
+static void schedule_ll_callback(void *data)
+{
+	const uint32_t begin_count = xthal_get_ccount();
```

Comment:
I would expect `xthal_get_ccount()` to actually go via Zephyr

Reply:
yep, we need to use Zephyr API here.

Reply:
Ah, you mean I should use functions from the Zephyr side like arch_timing_counter_get() ?

Reply:
sof_cycle_get_64